Tasks/AzureCLI/Strings/resources.resjson/ko-KR/resources.resjson

```diff
@@ -1,7 +1,7 @@
 {
   "loc.friendlyName": "Azure CLI preview",
   "loc.helpMarkDown": "Replace with markdown to show in help",
-  "loc.description": "Run a Shell script with Azure CLI commands against an azure subscription",
+  "loc.description": "Run a Shell or Batch script with Azure CLI commands against an azure subscription",
```

Comment:
We typically only checkin the en-US resource file.  We have a loc process that generates these and checks them in.  The server will fall back to en if it's only there.